oauth2client/client.py

```diff
@@ -2084,13 +2086,14 @@ def step2_exchange(self, code=None, http=None, device_flow_info=None):
                 token_response=d, scopes=self.scope,
                 token_info_uri=self.token_info_uri)
         else:
-            logger.info('Failed to retrieve access token: %s', content)
+            logger.info('Failed to retrieve access token: %s', response.data)
             if 'error' in d:
                 # you never know what those providers got to say
                 error_msg = (str(d['error']) +
                              str(d.get('error_description', '')))
             else:
-                error_msg = 'Invalid response: {0}.'.format(str(resp.status))
+                error_msg = 'Invalid response: {0}.'.format(
+                    str(response.status))
```

Comment:
Because this issue is all over `client.py`, and I'd prefer to fix all occurrences of it at once instead of doing piecemeal changes which lead to overall inconsistency.